test/e2e/storage/testsuites/provisioning.go

```diff
@@ -125,7 +129,10 @@ func (p *provisioningTestSuite) defineTests(driver TestDriver, pattern testpatte
 		l.config, l.testCleanup = driver.PrepareTest(f)
 		l.intreeOps, l.migratedOps = getMigrationVolumeOpCounts(f.ClientSet, dInfo.InTreePluginName)
 		l.cs = l.config.Framework.ClientSet
-		claimSize := dDriver.GetClaimSize()
+		tsSupportedSizeRange := p.getTestSuiteInfo().supportedSizeRange
+		claimSize, err := getSizeRangesIntersection(tsSupportedSizeRange, dDriver.GetClaimSizeRange())
+		framework.ExpectNoError(err)
```

Comment:
Use of `ExpectNoError` without explanation is an anti-pattern: https://github.com/kubernetes/kubernetes/issues/34059

Replace with:
```
framework.ExpectNoError(err, "determine intersection of test size range %+v and driver size range %+v", tsSupportedSizeRange, dDriver.GetClaimSizeRange())
```